diesel/src/mysql/connection/raw.rs

```diff
@@ -47,6 +47,15 @@ impl RawConnection {
         let port = connection_options.port();
         let unix_socket = connection_options.unix_socket();
 
+        #[cfg(not(windows))]
+        // allow single_match because compiler complains about if after cfg
+        // error: attributes are not yet allowed on `if` expressions
+        #[allow(clippy::single_match)]
+        match connection_options.ssl_mode() {
```

Comment:
Can we make `ConnectionOptions::ssl_mode()` just unconditionally return `None` on windows instead of having `cfg`'s all over the code?

Reply:
Is it convenient to do that this way?
3b9fd78aa9f878bb7552bb1ffa831c89365bfa82